pyproject.toml

```diff
@@ -41,6 +41,7 @@ dependencies = [
     "ruamel.yaml==0.18.6",
     "tabulate==0.9.0",
     "xmldict==0.4.1",
+    "reqstool-python-decorators ==0.0.1",
```

Comment:
New version 0.0.3.

The plugin also has a new version (0.0.2)

Looks good to me otherwise.